src/main/java/io/jenkins/plugins/appcenter/task/internal/DistributeResourceTask.java

```diff
@@ -99,8 +101,15 @@ private String parseReleaseNotesFromFile(@Nonnull String pathToReleaseNotes) {
 
     }
 
+    private static BuildInfo CreateBuildInfo(@Nonnull UploadRequest request) {
+        if (request.branchName == null && request.commitHash == null) return null;
+        return new BuildInfo(request.branchName,request.commitHash,null);
+    }
+
     @Override
     public PrintStream getLogger() {
         return taskListener.getLogger();
     }
+
```

Comment:
Just a small comment, but I think we should remove this blank space. At the end of this review I'll do this if there aren't other comments that I need your help with.